docs/hello-neighbour.yaml

```diff
@@ -381,15 +428,19 @@ paths:
           description: "User details"
           schema:
             $ref: "#/definitions/User"
+        401:
+          $ref: "#/responses/Unauthenticated"
+        403:
+          $ref: "#/responses/Unauthorized"
         404:
           description: "User not found"
           schema:
-            $ref:  "#/definitions/ErrorResponse"
+            $ref:  "#/definitions/Error"
     put:
       tags:
       - "member"
       summary: "Update user"
-      description: "Update user identified by user's UUID. All attributes can be edited."
+      description: "Update user. All attributes can be edited."
```

Comment:
I don't know if we necessarily want to allow updating all attributes (see above). Maybe, we shouldn't state it here for now.